File: ts/project/.sdk/tm/csharp/core/Control.cs
// ProjectName SDK - per-call control block.

namespace ProjectNameSdk;

public class Control
{
    public bool? Throw;
    public Exception? Err;
    public Dictionary<string, object?>? Explain;
    public string Actor = "";
    public Dictionary<string, object?>? Paging;

    // Outbound streaming marker: the entity `stream` method sets this to the
    // caller-supplied async payload so the request builder / transport can
    // stream it as the request body.
    public object? Stream_out;
}

File: ts/project/.sdk/tm/kotlin/core/Control.kt
package KOTLINPACKAGE.core

/** Per-call control state: throw behaviour, explain capture, actor, paging. */
class Control {

  // Tri-state: null (default: throw), true, or false (return fallback data).
  var throwing: Boolean? = null

  var err: RuntimeException? = null

  var explain: MutableMap<String, Any?>? = null

  var actor: String = ""

  var paging: MutableMap<String, Any?>? = null

  // Outbound streaming marker: the entity `stream` method sets this to the
  // caller-supplied streamable payload so the request builder / transport can
  // stream it as the request body.
  var streamOut: Any? = null
}
